programs/sample_http_link/README.md

```diff
@@ -0,0 +1,49 @@
+#Running Sample HTTP Link
+##Setup Identities
+###Create Identities
+We will create a host and client identity.
+
+For the purposes of this example and for the purpose of brevity we will create the device identities and configure the endpoints all in one command though this can be done separately.
+```
+ziti edge create identity device httpbin.server -a httpbinServerEndpoints -o httpbin.server.jwt
+ziti edge create identity device httpbin.client -a httpbinClientEndpoints -o httpbin.client.jwt
+```
+###Enroll The Identities
+We now have to enroll the identities so the controller knows about them.
+```
+ziti edge enroll httpbin.server.jwt
+ziti edge enroll httpbin.client.jwt
+```
+##Set Up The Service
+### Create A Service Config
+We'll create the service config before the actual service so that we can pass in the config when we create the service. This will save extra steps.
+```
+ziti edge create config httpbin-host.v1 host.v1 '{"protocol":"tcp", "address":"httpbin.org","port":80}'
+```
+###Create A Service
+Create a service to allow access the network traffic.
+```
+ziti edge create service httpbin --configs httpbin-host.v1
+```
+###Create Service Policies
+Lastly, we need to create service policies. The service policies define which identities can interact with certain services and how they interact with those services.  which edge routers are used to do so.
```

Comment:
the policies define what operations a given identity can perform (bind/dial). Maybe provide a link to the docs that explain what both do? 

I'd like to see the configs explained too pls?

"which edge routers are used to do so." seems like 'leftover' words?